src/neuroglancer/datasource/dvid/backend.ts

```diff
@@ -42,7 +42,12 @@ class VolumeChunkSource extends GenericVolumeChunkSource {
       // computeChunkBounds.
       let chunkPosition = this.computeChunkBounds(chunk);
       let {chunkDataSize} = chunk;
-      path = `/api/node/${params['nodeKey']}/${params['dataInstanceKey']}/raw/0_1_2/${chunkDataSize[0]}_${chunkDataSize[1]}_${chunkDataSize[2]}/${chunkPosition[0]}_${chunkPosition[1]}_${chunkPosition[2]}/nd`;
+ 
+      if (params['level'] == "0") {
```

Comment:
It seems this could be simplified by just setting params['dataInstanceKey'] to the level-specific data instance key.  Then there would not need to be any branching here.